_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "New series: From the editor's desk"
```

Comment:
For SEO purposes, can we include "OpenSearch" and "Inclusion" in the title? For example, "From the editors' desk: OpenSearch and Inclusion"